frontend/src/components/assessment-creation/EditAssessmentPopover.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  Button,
+  Divider,
+  IconButton,
+  Popover,
+  PopoverBody,
+  PopoverContent,
+  PopoverTrigger,
+  useDisclosure,
+  VStack,
+} from "@chakra-ui/react";
+import React from "react";
+import { MoreVerticalOutlineIcon } from "../../assets/icons";
+
+const EditAssessmentPopover = (): React.ReactElement => {
```

Comment:
let's rename this component to something else! we probably shouldn't call this `EditAssessmentPopover` since there is an actual `EditAssessment` option!